remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProgrammaticSearchMappingProviderImpl.java

```diff
@@ -22,6 +22,7 @@
    public void defineMappings(Cache cache, SearchMapping searchMapping) {
       searchMapping.entity(ProtobufValueWrapper.class)
             .indexed()
+            .indexName(cache.getName() + "--" + ProtobufValueWrapperAnalyzerDiscriminator.class.getName())
```

Comment:
Why not simply ``` .indexName(cache.getName())```? 
The index name is stored in each of entries of the lucene caches, so I'm a bit worried about large strings

Reply:
I just wanted to add some prefix or suffix to help distinguish it from possible other indexes that are user defined. A cache name usually comes from the user's domain so it is a potential conflict source. A shorter ```_protobuf``` suffix could work too.